src/chromeDebugAdapter.ts

```diff
@@ -19,6 +19,7 @@ import * as utils from './utils';
 import * as errors from './errors';
 
 import { FinishedStartingUpEventArguments } from 'vscode-chrome-debug-core/lib/src/executionTimingsReporter';
+import { isUndefined } from 'util';
```

Comment:
This is deprecated: https://nodejs.org/api/util.html#util_util_isundefined_object

Reply:
Fixed

Reply:
I don't see this change...